cmd/palomad/root.go

```diff
@@ -207,3 +218,13 @@ func rootPreRunE(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+var tempDir = func() string {
+	dir, err := os.MkdirTemp("", ".paloma")
+	if err != nil {
+		panic(fmt.Sprintf("failed creating temp directory: %s", err.Error()))
+	}
+	defer os.RemoveAll(dir)
```

Comment:
Is this call supposed to be here? We also remove try to remove it on line 45. But doesn't this just remove it after returning?

Reply:
It's a carbon copy of the the original code. Maybe it's not needed, and the calling code will recreate it? I'm not sure. I'd rather find a way to not use this hacky approach at all.